manim/mobject/geometry.py

```diff
@@ -737,6 +737,52 @@ def __init__(self, width=2, height=1, **kwargs):
 
 
 class AnnularSector(Arc):
+    """
+
+    Parameters
+    ----------
+    inner_radius : :class:`float`, optional
+       The inside radius of the Annular Sector.
+    outer_radius : :class:`float`, optional
+       The outside radius of the Annular Sector.
+    angle : :class:`float`, optional
+       The clockwise angle of the Annular Sector.
+    start_angle : :class:`float`, optional
+       The starting clockwise angle of the Annular Sector.
+    fill_opacity : :class:`float`, optional
+       The opacity of the color filled in the Annular Sector.
+    stroke_width : :class:`float`, optional
+       The stroke width of the Annular Sector.
```

Comment:
```suggestion
    inner_radius
       The inside radius of the Annular Sector.
    outer_radius
       The outside radius of the Annular Sector.
    angle
       The clockwise angle of the Annular Sector.
    start_angle
       The starting clockwise angle of the Annular Sector.
    fill_opacity
       The opacity of the color filled in the Annular Sector.
    stroke_width
       The stroke width of the Annular Sector.
```
The typings should be in the function parameters and not the docstrings.
See: https://docs.manim.community/en/latest/contributing/typings.html